CMakeLists.txt

```
@@ -65,7 +65,8 @@ endif()
 
 message(STATUS "OTBR package name: ${OTBR_NAME}")
 
-if(NOT OTBR_VERSION)
+set(OTBR_VERSION "" CACHE STRING "OTBR Package Version")
+if(OTBR_VERSION STREQUAL "")
```

```suggestion
if (NOT OTBR_VERSION)
```

I think this PR tries to use -DOTBR_VERSION="" to force reevaluating the package version.

But isn't the original code already support that?

@bukepo @suveshpratapa

---

I just wanted it to happen automatically when `./script/setup` is run, as that's the documented method to install or re-install the OTBR. So I made it part of that call chain.

---

Is the change of this PR in `script/_otbr` already enough to fulfil the purpose?
I am trying to understand if the changes to `CMakeLists.txt` is necessary.

---

@simonlingoogle You're right. I wanted to make it be the same as `OT_PACKAGE_VERSION` in `third_party/openthread/CMakeLists.txt`, but it appears it wasn't necessary.  The change in `script/_otbr` is enough.  I reflected the change in the pull request, please take a look.